spec/xclarity_client_switch_spec.rb

```diff
@@ -94,4 +94,37 @@
       end
     end
   end
+
+  describe 'GET /switches/UUID' do
+    context 'attributes included in the version 1.4' do
+      it 'getting new attributes' do
```

Comment:
I would like to ask you organize these test better. It is important to split it in contexts and being some behavioral tests(e.g.: check values structure). Once having a possible data structure changes in the future API versions, is it important track it and identify it in tests.